CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ endif()
 include(FetchContent)
 set(_itk_build_testing ${BUILD_TESTING})
 if(WASI OR EMSCRIPTEN)
+  # Cleaver lib vendored GTest incompatibility
   set(BUILD_TESTING OFF CACHE BOOL "Build testing code" FORCE)
 endif()
```

Comment:
```suggestion
# Testing
if(DEFINED BUILD_TESTING)
  set(_itk_build_testing ${BUILD_TESTING})
else()
  set(_itk_build_testing OFF) # Default
endif()
# Since Cleaver internally adds an external project downloading googletest
# that would conflict with the one provided by ITK, we systematically prevent
# Cleaver test from being configured & built.
set(BUILD_TESTING OFF CACHE BOOL "Build testing code" FORCE)
```

Reply:
The Cleaver2 lib vendored GTest incompatibility is only with the wasm toolchains. For other builds, we do want to be able to build/enable the ITK tests.

Reply:
I incorporated the first part of this improvement:

```
if(DEFINED BUILD_TESTING)
  set(_itk_build_testing ${BUILD_TESTING})
else()
  set(_itk_build_testing OFF) # Default
endif()
```

In to the other committed suggestion.